examples/achromatic_spectrometer/README.rst

```diff
@@ -0,0 +1,53 @@
+.. _examples-spectrometer:
+
+Achromatic Spectrometer
+========================
+
+A spectrometer beamline using a bending dipole.  A transversely-tapered plasma lens is added for chromatic correction.  The tapered plasma
+lens design is based on:
+
+C. A. Lindstrom, presentation at the EuroNNAc Special Topics Workshop 2022:
+"Solutions and challenges for a multi-stage plasma accelerator",
+
+https://agenda.infn.it/event/28376/contributions/178724/attachments/96899/133588/Lindstr%C3%B8m,%20EuroNNAc%20workshop,%2022%20Sep%202022.pdf
+
+We use a 1 GeV electron beam with initial normalized rms emittance of 2 microns.
```

Comment:
Let's mention also the initial energy spread?
(Wait, there is none?)

Reply:
There is--it is 2%.